contribute_to_pim/front.rst

```diff
@@ -0,0 +1,152 @@
+
+    rm -rf var/cache
+    bin/console --env=prod pim:installer:dump-require-paths
+    yarn webpack-dev
+
+Conclusion
+++++++++++
+
+With those commands you now know what to do exactly when you modify some files on the PIM. But most of the time there are more efficient way to do.
+
+.. _QuickAndEasyWay:
+
+The quick and easy way
+----------------------
+
+Most of the time, when you are contributing to the PIM you do a bit of everything at the same time. Sometimes, it can be hard to keep track of which command to run and when. That's why it can be really handy to define an alias in your terminal to run a compilation of them.
+
+If you touch a configuration file, a .less file or a translation file (anything but a javascript file)
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+run this command:
+
+.. code-block:: bash
+
+    rm -rf ./var/cache/*; rm -rf ./web/js/*; rm -rf ./web/css/*; /usr/local/opt/php\@7.2/bin/php bin/console pim:installer:dump-extensions --env=prod; /usr/local/opt/php\@7.2/bin/php bin/console pim:install:ass --env=prod --symlink;
+
+We strongly advise you to create an alias
+
+If you only modify a javascript file
+++++++++++++++++++++++++++++++++++++
+
+.. warning::
+
+    If you are only working on javascript files, you don't need to run the previous command. Never
+
+Instead, you can simply run
+
+.. code-block:: bash
+
+    yarn webpack-watch
+
+This will run the initial build of webpack and then recompile each time you modify a javascript file (and reload your browser).
+
+What webpack command to run?
+----------------------------
+
+Akeneo PIM provides three webpack commands to build the javascript artefacts
+
+yarn webpack
+++++++++++++
+
+This first command will build the javascript file for production. The javascript will be minified and this process can take a lot of time. It's not advised to use this command in development phase.
+
+yarn webpack-dev
+++++++++++++++++
+
+This command will build the javascript artefacts in development mode. The size of the generated bundle will be higher and quicker to generate. It's the prefered way to rebuild the frontend after checkouting another branch when you are not actively working on the frontend.
+
+yarn webpack-watch
+++++++++++++++++++
+
+>This command does exactly what the yarn webpack command does but will not exit at the end of the process.
+Instead, it will wait for modifications and recompile the changed files if needed. It will then reload your browser to see the modifications in the PIM.
+
+Last word about the browser cache
+---------------------------------
+
+We currently don't manage dynamic assets filenames to automatically force browser cache update. So you will need to clear your browser cache when working on the PIM frontend. What we advise to avoid this problem is to disable the browser cache when your debug console is open (this option is available on the main browser in the market).
```

Comment:
debug console is open*ed*

Reply:
we can say chrome, it's not a dirty word right :stuck_out_tongue: